llvm/lib/Target/X86/X86.td

```diff
@@ -66,6 +66,9 @@ def FeatureXSAVES  : SubtargetFeature<"xsaves", "HasXSAVES", "true",
                                        "Support xsaves instructions",
                                        [FeatureXSAVE]>;
 
+def FeatureBIDENCODING : SubtargetFeature<"bid-encoding", "HasBIDENCODING", "true",
+                                          "Support BID encoding for decimal floatin-points">;
```

Comment:
Per other comments, I think we need to figure out how to make this not be X86 specific.

Also, typo: "decimal floatin-points" -> "decimal floating-point".